### ISSUE TITLE
[Kommentierung 1.5.3] KOM-LE-A_2187-05 - Authentifizierung des KOM-LE-Teilnehmers über AUT-Zertifikat am AccountManager

### ISSUE THREAD

**Issue by CLAM01** [gematik/api-kim#44]
![image](https://github.com/gematik/api-kim/assets/115494634/09c7a5f4-f176-4dc5-914f-965e4d1fce7d)

Kann der Satz "Zur Pflege der Basisdaten des Verzeichnisdienstes und..." kann m.E. endlich raus, da wir keine Basisdaten mehr pflegen.

**Comment by dhufnagel** [comment]
Wie sieht es hierbei mit der Rückwärtskompatibilität aus? Kann ein CM mi 1.5 sich auch an einem Fachdienst mit 1.5.3 authentifizieren, wenn nur nbf gesetzt ist?

**Comment by CLAM01** [comment]
Ein CM das nur nbf hat, sollte dann nicht mehr gehen. Der Fachdienst 1.5.3 kennt dann nbf nicht mehr.

**Comment by gem-cp** [comment]
Den Teil mit "Pflege der Basisdaten" werden wir entfernen.
Zu nbf: Wir brauchen Abwärtskompatibilität. Daher werden wir für den Fachdienst fordern, dass er das Alte Token und das neue Token unterstützen muss.

**Comment by dhufnagel** [comment]
Ich würde dringend den Vorschlag von Herrn Stucke unterstützen. Es ist unüblich und sicherheitstechnische bedenklich, wenn der Client die Gültigkeitsdauer vorgibt.

**Comment by CLAM01** [comment]
Das sollte dann aber auch irgendwo in der Spezifikation festgehalten werden als Nebensatz / ergänzende Info unter der Afo.

**Comment by CLAM01** [comment]
Die Gültigkeitsdauer ist doch auf 6 Stunden begrenzt laut Afo oder?

**Comment by gem-cp** [comment]
OK. Danke für das Feedback. Wir werden die Änderung am Token zurücknehmen. Das Token bleibt so wie bisher in 1.5.2.

**Comment by Amarteau** [comment]
Wird die Gültigkeit des Tokens trotzdem allgemein über die Spezifikation definiert werden?
Oder anders gefragt, woher weiß das Clientmodul ob ein vorhandener JWT noch gültig ist oder ein neuer JWT erzeugt werden muss?

**Comment by Amarteau** [comment]
Ja, aber die 5 Minuten sind ja nur der empfohlene Standardwert laut Tab_Konfig_Parameter. Ein Fachdienst könnte den Wert ja auch auf 2 Minuten konfigurieren.
Aus IOP Sicht würde ich es begrüßen, wenn man den Wert analog der Passwortpolicy vom Fachdienst abfragen kann.

**Comment by gem-cp** [comment]
Wir können die Gültigkeitsdauer über ServiceInformation.yaml abfragbar machen. Gibt es Gegenstimmen?

**Comment by Amarteau** [comment]
Ja, kann ich machen.

**Comment by Amarteau** [comment]
https://github.com/gematik/api-kim/pull/54

**Comment by gem-cp** [comment]
Danke, werden wir übernehmen; minimum wird auf 300 geändert.
